src/views/general-info.js

```diff
@@ -0,0 +1,44 @@
+import React, {Component} from 'react'
+import MainMenu from './main-menu'
+import GraphSet from '../components/graph-set'
+import axios from 'axios'
+
+class AllTeamsInfo extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      data: undefined
+    }
+    axios.get('/api/team/info/all/')
+      .then(info => {
+        this.setState({data: info.data})
+      })
+      .catch(err => {
+        alert('error is ' + JSON.stringify(err))
+      })
+  }
+
+  render () {
+    if (this.state.data) {
+      return (
+        <div className="text-center" style={{
+          margin: '20px'
+        }}>
+          <MainMenu view="general-info"/><br/>
+          <GraphSet config="" parser={a => {
```

Comment:
Add parser and config, this is highly unorganized.

Reply:
I agree, no need for config, just a "parser". most of the parsing happens on the server side...

Reply:
So why do you need a parser and not a config?